tensorflow_lite_support/python/test/task/text/text_searcher_test.py

```diff
@@ -260,7 +260,7 @@ def test_search_with_default_options(self):
     text_search_result = searcher.search('The weather was excellent.')
 
     self.assertProtoEquals(_EXPECTED_REGEX_DEFAULT_OPTIONS_SEARCH_RESULT,
```

Comment:
Please convert the definition of expected results from proto text to defining them using SearchResult objects. Ideally you should compare the expected results with computed results using `self.assert()`. If that doesn't work because of floating point accuracy issue, you can fall back to using `.to_pb2()` and compare using `self.assertProtoEquals` like what you're doing now.

Reply:
@khanhlvg I think we'll have to fall back to using `.to_pb2()` as it seems to compare at full precision instead of `1e-6` like how `assertProtoEquals` is able to do right now. Meanwhile, I've updated the expected result objects to use the SearchResult python objects instead of the protobuf text protos.

Reply:
Falling back to assertProtoEquals and .to_pb2() SGTM. Thanks for checking.